src/pages/resultsView/download/CaseAlterationTable.tsx

```diff
@@ -192,29 +199,53 @@ export default class CaseAlterationTable extends React.Component<ICaseAlteration
             }
         ];
 
-        const typeSet : {[x: string]: boolean} = {};
+        const geneSet : {[x: string]: OQLLineFilterOutput<AnnotatedExtendedAlteration>} = {};
         this.props.oqls.forEach(oql => {
+            if (oql.gene in geneSet) {
+                geneSet[oql.gene].oql_line += oql.oql_line;
+            }
+            else {
+                geneSet[oql.gene] = oql;
+            }
+        });
+        
+        _.values(geneSet).forEach(oql => {
```

Comment:
i don't think you need to first call _.values.  _.forEach(geneSet, ()=>)  will do it without extra iteration cycle.  here, impact is negligible but in general ...